conf.yaml

```diff
@@ -523,6 +523,25 @@ contents:
                 path:   docs/shared
                 exclude_branches: [ 1.0 ]
 
+         -
+            title:      Elastic Cloud on Kubernetes (ECK)
```

Comment:
Do we want to include `(ECK)` in the title here? I would imagine we probably want to just use this in the docs themselves.

Reply:
Done.